content/sensu-go/6.0/reference/entities.md

```diff
@@ -66,6 +66,32 @@ You can modify the proxy entity later if needed.
 
 Use [proxy entity filters][19] to establish a many-to-many relationship between agent entities and proxy entities if you want even more power over the grouping.
 
+## Create and modify agent entities
+
+When an agent connects to a backend, the agent entity defnition is created from the information in `/etc/sensu/agent.yml`.
```

Comment:
I would maybe say "is created from the agent.yml config file," with a link to where the default file location is on each OS. ex. linux: `/etc/sensu/agent`, freebsdb: `/usr/local/etc/sensu`, windows:`C:\ProgramData\sensu\config` or something like that?

Reply:
I'm only including the OS platforms we have supported packages for